### PR TITLE
Start RTCP package: add header parsing

### PR DESCRIPTION
**Pull request by maxhawkins**
More to come. Let me know if there's anything I need to change otherwise I'll proceed with the rest of the packet types.

### CODE REVIEW

**Comment by Sean-Der** [comment]
Hey @maxhawkins fantastic work!

Mind branching off of master instead (rtcp branch has C code so won't build in CI) so we don't get the tests.

File: pkg/rtcp/packet.go [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+
+import (
+	"encoding/binary"
+
+	"github.com/pkg/errors"
+)
+
+// RTCP packet types registered with IANA. See: https://www.iana.org/assignments/rtp-parameters/rtp-parameters.xhtml#rtp-parameters-4
+const (
+	TypeSR   = 200 // sender report        RFC 3550, 6.4.1
```
Comment (Sean-Der) [inline pkg/rtcp/packet.go]: If these stay public can you make them the full name `TypeSenderReport` if they stay private it is fine using a less descriptive name (I just want to ensure the GoDoc isn't confusing)

I think they will end up being private if you do an API like.
```
pkt := &rtcp.Nack{SequenceNumber: 5000,  SSRC: 1234}
raw, err := pkt.Marshal()
```

Comment (maxhawkins) [reply pkg/rtcp/packet.go]: Not sure what the API is going to look like yet. I'll make them public and unabbreviated for now.

**Comment by maxhawkins** [comment]
OK, PTAL

**Comment by maxhawkins** [comment]
OK I think this is ready to merge.

**Pull request merged**